.changeset/tender-onions-leave.md

```diff
@@ -0,0 +1,7 @@
+---
+'@makeswift/runtime': minor
+---
+
+Adds a slimmer rich text component for live pages so that large dependencies like Slate and Immutable aren't included in bundles for live pages and are only used in the Makeswift builder. This reduces the overhead of the Makeswift runtime for live pages and boosts performance.
+
+While behavior is intended to be the same, these changes modify the structure of the DOM for live pages, which could cause issues with existing sites if they're relying on the DOM structure of the Text component. For this reason we're releasing this in a minor update as a _breaking change_.
```

Comment:
Is there anything that should clue a user into this being a breaking change other than the fact that we are calling it out? 

I am trying to understand why we are releasing a breaking change as a minor version as opposed to a major version. If you have a link to this please share.

Reply:
It's general practice in SemVer, that before a 1.0.0 release minor changes are breaking changes and patch changes are more or less like minor changes. For example, the `^` caret range specifier respects this de-facto standard by treating pre-1.0.0 versions differently. See [this](https://stackoverflow.com/a/22345808) StackOverflow answer. In essence, `0.6.0` is _not_ compatible with `^0.5.0`, but `1.6.0` _is_ compatible with `^1.5.0`

Reply:
😮